### PR TITLE
Persist property values parsed from BuildConfiguration

### PR DESCRIPTION
**Pull request by ericstj**
We need the TargetGroup/OSGroup/etc properties that make
up BuildConfiguration for binplacing.  Persist these
as properties prefixed with `_bc_`.

/cc @weshaggard @tarekgh @joperezr

### CODE REVIEW

File: src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs [hunk 1]
```diff
@@ -246,13 +253,19 @@ private void CreateBuildConfigurationPropsFile(Configuration buildConfiguration)
...
         {
             var propertyGroup = parent.ContainingProject.CreatePropertyGroupElement();
             parent.AppendChild(propertyGroup);
 
             propertyGroup.AddProperty(value.Property.Name, value.Value);
 
+            if (!String.IsNullOrEmpty(parsedValuePrefix))
+            {
+                propertyGroup.AddProperty(parsedValuePrefix + value.Property.Name, value.Value);
+
```
Comment (joperezr) [inline src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs]: NIT: extra line 😄

**Comment by tarekgh** [comment]
LGTM

**Comment by ericstj** [comment]
@dotnet-bot help

**Comment by ericstj** [comment]
@dotnet-bot skip ci please

**Comment by ericstj** [comment]
merging as this passed CI and I only removed a single line of whitespace.

**Pull request merged**